src/Init/Data/Int/Basic.lean

```diff
@@ -325,6 +325,19 @@ protected def pow (m : Int) : Nat → Int
 instance : HPow Int Nat Int where
   hPow := Int.pow
 
+protected
+theorem pow_succ (b : Int) (n : Nat) : b ^ (n+1) = b^n * b := by rfl
```

Comment:
```suggestion
protected theorem pow_succ (b : Int) (n : Nat) : b ^ (n+1) = b^n * b := by rfl
```

Reply:
pow_succ has also been introduced in a later file in the meantime! (I'm inclined to believe it belongs later than here, this file should only have the minimal theorems needed for definitions.)